dev/com.ibm.ws.appclient.boot/test/com/ibm/ws/kernel/boot/internal/commands/SharedBootstrapConfig.java

```diff
@@ -61,7 +61,7 @@ private SharedBootstrapConfig(String serverName) throws IOException {
 
         HashMap<String, String> map = new HashMap<String, String>();
 
-        this.findLocations(serverName, rootDirStr, null, null, null);
+        this.findLocations(serverName, rootDirStr, null, null, null, null);
```

Comment:
Out of curiosity, I assume this is not really needed because the original method just calls this method with null value.

Reply:
The change is unnecessary.  I'll back it out.